packages/css/markprompt.css

```diff
@@ -37,6 +38,24 @@
   }
 }
 
+/* Remove the 'x' clear button from the search input field */
+input[type='search']::-ms-clear {
+  display: none;
+  width: 0;
+  height: 0;
+}
+input[type='search']::-ms-reveal {
+  display: none;
+  width: 0;
+  height: 0;
+}
+input[type='search']::-webkit-search-decoration,
+input[type='search']::-webkit-search-cancel-button,
+input[type='search']::-webkit-search-results-button,
+input[type='search']::-webkit-search-results-decoration {
+  display: none;
+}
+
```

Comment:
This will remove the clear button from all search input fields on sites where this CSS is loaded, we should probably use a more specific selector like `.MarkpromptPrompt`?

Reply:
Indeed